Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -303,9 +304,12 @@ void SetLastTransform(const CGAffineTransform* transform) {
         return &lastTransform;
     }
 
-    void AllowAllExceptFinalEndFigure(bool allow) {
-        SetAllowsFigureCalls(allow);
-        geometrySink->AllowAllExceptFinalEndFigure(allow);
+    HRESULT AllowAllExceptFinalEndFigure(bool allow) {
+        HRESULT hr = SetAllowsFigureCalls(allow);
```

Comment:
Oh... yeah. I don't know why I did it that way when I've used the macros in every other conceivable instance.